src/main/resources/assets/css/main.less

```diff
@@ -570,6 +570,43 @@ fieldset[disabled] .btn-info.active {
   background-color: #8e929b;
 }
 
+.health-bar-column {
+  vertical-align: middle;
+
+  .health-bar {
```

Comment:
I missed, that this bar was actually in the design from the beginning. Could we add it in there? There was some design choices behind it. I'll let @aatumanova explain